ui/scss/_notification.scss

```diff
@@ -0,0 +1,156 @@
+@import 'variables';
+.top-navbar__notification, .all-notification {
+    .all-notification__view,li {
+        min-width: 350px;
+        a {
+            padding: 0px 4px;
+            span {
+                display: inline-block;
+                width: 80%;
+                @media screen and (max-width: 1200px) {
+                    width: 70%;
+                }
+            }
+        }
+        border-bottom: solid;
+        border-color: #C4C4C4;
+        border-width: 1px;
+        text-align: left;
+        #notification-header:hover {
+            background-color: #fff
+        }
+    }
+    .notification__avatar{
+        height: 35px;
+        width: 35px;
+        border-radius: 50%;
+        float: left;
+        -webkit-border-radius: 50%;
+        -moz-border-radius: 50%;
+        -ms-border-radius: 50%;
+        -o-border-radius: 50%;
+    }
+    .notification__time {
+        color: #C4C4C4;
+        padding-left: 5px;
+    }
+    .notification-unread{
+        background-color: #dde1f8;
+        .notification__time {
+           color: rgb(172, 172, 172)
+        }
+    }
+    .notification-footer {
+        text-align: center;
+    }
+}
+.dropdown {
+    .notification-header:hover {
+        background-color: #fff;
+    }
+    .notification-header {
+        padding-left: 0px ;
+        padding-right: 0px ;
+        margin: 0px;
+        text-align: center;
+    }
+}
+.badge {
+    background-color: red;
+    padding: 5px;
+    border-radius: 25px;
+    -webkit-border-radius: 25px;
+    -moz-border-radius: 25px;
+    -ms-border-radius: 25px;
+    -o-border-radius: 25px;
+    position: absolute;
+    top: 0;
+}
+.all-notification {
+    display: flex;
+    flex-direction: column;
+    font-size: 14px;
+    .all-notification__avatar {
+        height: 50px;
+        border-radius: 50%;
+        -webkit-border-radius: 50%;
+        -moz-border-radius: 50%;
+        -ms-border-radius: 50%;
+        -o-border-radius: 50%;
+        float: left;
+        padding: 5px;
+    }
+    .all-notification__view:hover {
```

Comment:
@Nziranziza You can use `&:hover` selector instead of `.all-notification__view:hover`